server/commands/teamCreator.ts

```diff
@@ -33,7 +35,9 @@ async function teamCreator({
   ip,
 }: Props): Promise<TeamCreatorResult> {
   let authP = await AuthenticationProvider.findOne({
-    where: authenticationProvider,
+    where: id
+      ? { ...authenticationProvider, teamId: id }
```

Comment:
Note there is currently no index on `teamId` – need to add this now we're querying it

Reply:
I don't think performance is any worse than before — the addition of teamId should be applied after the query planner makes use of existing indices

Reply:
I suppose it still has the providerId, ok 👍